.github/dependabot.yml

```diff
@@ -5,7 +5,8 @@
 
 version: 2
 updates:
-  - package-ecosystem: 'npm'
+  - target-branch: 'maintenance/v1.x'
```

Comment:
Created two separate dependabot configs for the 1.x and the development branches.